spec/factories/users.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  sequence :email do |n|
+    "test#{n}@example.com"
+  end
+
+  factory :user do
+    email { generate :email }
+    password 'asdfasdf'
+    password_confirmation 'asdfasdf'
+    confirmed_at Time.current
+  end
```

Comment:
@marderer let's use Faker for all generated data. Purpose: check that tests pass with different generated data